test/e2e/network/service.go

```diff
@@ -2491,7 +2491,11 @@ var _ = common.SIGDescribe("Services", func() {
 		// assert 5 times that both the local and remote pod can connect to the Service while all endpoints are terminating
 		serviceAddress := net.JoinHostPort(svc.Spec.ClusterIP, strconv.Itoa(servicePort))
 		for i := 0; i < 5; i++ {
-			// both first and second pause pod can connect to the terminating webserver since iTP=Cluster
+			// There's a terminating pod called webserver0 running on node0.
+			// There's a Service with internalTrafficPolicy=Cluster
+			// pausePod0 and pausePod1 are on node0 and node1 respectively.
+			// pausePod0 -> Service clusterIP succeeds because traffic policy is "Cluster"
+			// pausePod1 -> Service clusterIP succeeds because traffic policy is "Cluster"
```

Comment:
Probably this makes more sense if you've been reading the code up to here, but it's a little confusing with no further context... Maybe  change the first two lines to something like "There's a Service with internalTrafficPolicy=Cluster, with a single endpoint (which is terminating) called webserver0 running on node0." Likewise below.

You could maybe not bother to name webserver0 and just refer to it as "the endpoint" instead...

(LOL, complaining this much about _comments_ is classic bikeshedding, but since they are new comments that are explicitly being added to try to clarify the code I feel a little less bad about it...)

Reply:
Thanks, updated the wording a bit here